### PR TITLE
add nysdoh_nursinghomes.py and sql

### PR DESCRIPTION
**Pull request by baolingz**

### CODE REVIEW

File: etl/nysdoh_nursinghomes.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                                operation=lambda row: get_sname(row['address'])
+                                    )
+                        ]),
+
+    geo_flow,
+    add_computed_field([dict(target=dict(name = 'the_geom', type = 'string'),
+                            operation=lambda row: get_the_geom(row['geo_longitude'], row['geo_latitude'])
+                            ),
+                        ]),
+    delete_fields(fields=['boro']),
```
Comment (SPTKL) [inline etl/nysdoh_nursinghomes.py]: don't delete boro

File: etl/nysdoh_nursinghomes.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    add_field('datasource', 'string', table_name),
+
+    ################## geospatial ###################
+    ###### Make sure the following columns ##########
+    ###### exist before geo_flows          ########## 
+    #################################################
+
+    rename_field('zip_code','zipcode'),
+    
+    add_computed_field([dict(target=dict(name = 'boro', type = 'string'),
```
Comment (SPTKL) [inline etl/nysdoh_nursinghomes.py]: can you add a boro conversion function in utils? 
e.g. `convert_to_boro("New York") = 'MN'`, `convert_to_boro("Richmond") = 'SI'`
and include additional options such as `convert_to_boro(1) = 'MN'`,
``convert_to_boro("X") = 'BX'`
ideally every dataset would have the boro field, even if it has to be empty

Comment (baolingz) [reply etl/nysdoh_nursinghomes.py]: @SPTKL just updated it as you suggested.

**Comment by SPTKL** [comment]
@baolingz fix syntax error
```
psql:/home/db-facilities/sql/nysdoh_nursinghomes.sql:28: ERROR:  syntax error at or near "    ADD"
LINE 2:     ADD hash text, 
        ^
psql:/home/db-facilities/sql/nysdoh_nursinghomes.sql:56: ERROR:  syntax error at or near ""bed_type/service_category""
LINE 5:                 WHEN "bed_type/service_category" = 'Total Re...
```

**Pull request closed without merging**